src/components/userPage/components/SideMenu.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Menu, Icon } from 'antd';
+
+import { USER_PAGE } from './constants';
+
+const { 
+    CLASSNAMES: { MENU },
+    STRINGS: { DARK, FOLDER, INFO_CIRCLE, INLINE, MESSAGE, 
+        NOTIFICATION, SETTING, SHOPPING, SOLUTION },
+    TEXTS: { ACCOUNT_INFO, MESSAGES, HELP, NOTIFICATIONS, 
+        PURCHASES, SAVED_ITEMS, SETTINGS },
+} = USER_PAGE;
```

Comment:
fix the destructing of `STRINGS` AND `TEXTS`

Reply:
done